### PR TITLE
(READY FOR TEST-MERGING) Full Medical Remap + Virology part 1

### PR DESCRIPTION
**Pull request by Kaksisilma**
Remapping medical from scratch. Heavily inspired by Bay's but better because we'll have virology 👍

This first part will only contain the map and a decomissioned Virology area. Proper Virology will be added later on another PR for ease of reviewing.

TODO:
- Polish
- Add Virology in a separate PR

<details> <summary> Current progress as of 13/03/2022 (DD/MM/YYYY) </summary>

First Deck:
Capture3

Second Deck:
Capture2

 </details>

### CODE REVIEW

File: maps/torch/torch6_bridge.dmm [hunk 1]
```diff
@@ -19185,6 +19185,10 @@
 	d2 = 2;
 	icon_state = "1-2"
 	},
+/obj/machinery/turretid/tur_ID/dagon{
```
Comment (Kaksisilma) [inline maps/torch/torch6_bridge.dmm:19195]: Moved the exterior turret controller to the bridge to free up some space near robotics. Also makes more sense there.

**Comment by ProbablyCarl** [comment]
Genuinely incredible. Thank you.
I hope you're aware that the issue with Virology was rads giving space aids immediately, regardless of how much. With that sorted, we're golden. Not sure what you changed otherwise.

**Comment by ProbablyCarl** [comment]
Keep in mind, pod removal is a PITA and you'll have to make sure they're all numbered correctly. Map wise, that is.

**Comment by Kaksisilma** [comment]
Tested locally. All related bugs patched. This is ready for a test-merge.

**Comment by ProbablyCarl** [comment]
Test merging.

**Comment by Kaksisilma** [comment]
This first test went well. Here's the main complaints I got:
1 - Cloning room is too far away from the ETC
2 - Triage room is useless
3 - The lift is useless
4 - One of the maintenance ladders was erased
5 - No glass and steel near the autolathe
6 - Only one chem dispenser
7 - Bugs relating to the buttons and the robotics-medical airlock

I have changed the layout a bit and was able to fix 7, 5, 4 and 2.
Issue 6 is controversial, and some people argue in favor of the one chem machine setup and others argue against it.
Issue 3 is noted, but the lift feels like a necessity with the mech charging port downstairs, and splitting the morgue on two different rooms is not very interesting.
Issue 1 is agreeable, and by far the hardest to fix because of the position of the saferoom. There are no good spots to put the saferoom at (Even if we consider the deck under it) and the option of removing it and expanding the morgue is not very interesting.

The voting feedback was generally favorable, at 24%, 16%, 18%, 42% for, respectively: "I liked it; I am indifferent; I disliked it; I did not play on the Dagon or did not visit the medbay."

Overall, I'd say that this can be merged after a few more days of testing if no other bugs show up, and Issues 6, 3 and 1 can be worked on with time.

**Comment by anconfuzedrock** [comment]
for 3, first off,  Ithink mechs can use ladders. second off, why the FUCK does medical need a mech charger? robotics is right nextdoor. and even if it does, just put the charger upstairs instead of a big ass elevator.

**Comment by IrkallaEpsilon** [comment]
2 chem dispensers down to 1. 2 Body scanners down to 1? Ugh.

Nvm saw the combat medkits. Thats good

**Pull request merged**